src/Symfony/Component/Messenger/Tests/Transport/AmqpExt/AmqpExtIntegrationTest.php

```diff
@@ -206,4 +221,25 @@ private function createSerializer(): SerializerInterface
             new SerializerComponent\Serializer([new ObjectNormalizer(), new ArrayDenormalizer()], ['json' => new JsonEncoder()])
         );
     }
+
+    private function assertApproximateDuration($startTime, int $expectedDuration)
+    {
+        $actualDuration = microtime(true) - $startTime;
+
+        $this->assertEquals($expectedDuration, $actualDuration, 'Duration was not within expected range', .5);
```

Comment:
The delta argument is deprecated in phpunit 8. Maybe we should check if `assertEqualsWithDelta` exist as use this instead. See https://github.com/sebastianbergmann/phpunit/issues/3341